udf-server-python/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.aklivity.zillabase</groupId>
+    <artifactId>zillabase</artifactId>
+    <version>develop-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>udf-server-python</artifactId>
+  <name>zillabase::udf-server-python</name>
+
+  <properties>
+    <jacoco.coverage.ratio>1.0</jacoco.coverage.ratio>
+    <jacoco.missed.count>0</jacoco.missed.count>
+  </properties>
+
```

Comment:
Shouldn't we be triggering a python "compile" / "lint" from the maven build here?